df.viewscreen.xml

```diff
@@ -1562,11 +1564,11 @@
         <int32_t name='supply_column'/>
         <int32_t name='display_row'/>
         <int8_t name='show_play_now'/>
-        <stl-vector type-name='int8_t'/>
-        <stl-vector type-name='int32_t'/>
+        <stl-vector name='choice_types' type-name='int8_t' comment='0 - play now, 1 - prepare, 2 - profile'/>
```

Comment:
This could probably be an enum (untested):
```xml
<stl-vector name='choice_types'>
  <enum base-type='int32_t'>
    <enum-item name='PlayNow'/>
    <enum-item name='Prepare'/>
    <enum-item name='Profile'/>
  </enum>
</stl-vector>
```